packages/dd-trace/test/appsec/iast/analyzers/path-traversal-analyzer.spec.js

```diff
@@ -134,6 +134,26 @@ describe('path-traversal-analyzer', () => {
     expect(addVulnerability).to.have.been.calledOnce
     expect(addVulnerability).to.have.been.calledWithMatch(iastContext, { evidence: { value: 'taintedArg2' } })
   })
+
+  it('Should not report the vulnerability if it comes from send module', () => {
+    const proxyPathAnalyzer = proxyquire('../../../../src/appsec/iast/analyzers/path-traversal-analyzer', {
+      './injection-analyzer': InjectionAnalyzer,
+      '../iast-context': { getIastContext: () => iastContext }
+    })
+
+    proxyPathAnalyzer._getLocation = function () {
+      return { path: 'node_modules/send/send.js', line: 3 }
+    }
+
+    addVulnerability.reset()
+    TaintTrackingMock.isTainted.reset()
+    getIastContext.returns(iastContext)
+    TaintTrackingMock.isTainted.returns(true)
+    hasQuota.returns(true)
+
+    proxyPathAnalyzer.analyze(['arg1'])
+    expect(addVulnerability).to.be.callCount(0)
```

Comment:
More used in our code
```suggestion
    expect(addVulnerability).not.to.be.called
```

Reply:
Might `to.not.have.been.called` is more suitable in this case

Reply:
Agree.